.devcontainer/post_create_command.sh

```diff
@@ -1,2 +1,7 @@
 # ensure pre-commit is installed
 pre-commit install
+
+# run yarn install in docs folder
+cd docs
+yarn install
+cd ..
```

Comment:
maybe we can leave this away? how much time does it add?

Reply:
Takes a few seconds, maybe worth having in default container to make it super easy for people to build docs site in a codespace if they want. Of course all they read need do is change a .md file and the workflow will build or fail so if not like they need to. 

But maybe handy to have there to make docs contributions as frictionless as possible.